internal/services/mysql/registration.go

```diff
@@ -65,7 +65,6 @@ func (r Registration) SupportedResources() map[string]*pluginsdk.Resource {
 	}
 
 	if !features.FourPointOhBeta() {
-		resources["azurerm_mysql_active_directory_administrator"] = resourceMySQLAdministrator()
 		resources["azurerm_mysql_configuration"] = resourceMySQLConfiguration()
```

Comment:
What about all of these resources that were removed in 4.0?

Reply:
Thanks for your time. Other resources have also been removed. Could you please take another look?